layouts/blog/content.html

```diff
@@ -0,0 +1,25 @@
+<div class="td-content">
+	<h1>{{ .Title }}</h1>
+	{{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
+	<div class="td-byline mb-4">
+		{{ with .Params.author }}{{ T "post_byline_by" }} <b>{{ . | markdownify }}</b> |{{ end}}
+    {{ if .Params.translator }}
+      Translated {{ with .Params.translator }}{{ T "post_byline_by" }} <b>{{ . | markdownify }}</b> |{{ end }}
+    {{ end }}
```

Comment:
Both `{{ if }}` and `{{ with }}` are used to check and access `.Params.translator`, but they are redundant in this context. For a more optimized and efficient code, please use only one of them.

Below are suggestions below for reference if they help, please feel free to refer to them.

In the [en.toml](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml) file add the `translated_by` 
_(I think using the value ‘Translated by’ would be more suitable for localization.)_

```toml
[translated_by]
other = "Translated By"
```

Then use that `translated_by` in the layout file here.

```suggestion
      {{ with .Params.translator }}{{ T "translated_by" }} <b>{{ . | markdownify }}</b> |{{ end }}
```

Reply:
Thanks, fixed by https://github.com/kubernetes/website/pull/47270/commits/3444b2616332c0da9d2b0c9366c99d5b6698f1ef!